infra/teardown.md

```diff
@@ -0,0 +1,96 @@
+some guidelines when tearing down Terraform namespaces:
+
+* Tear down accounts that don't hold resources for organization-wide
+  purposes first -- leave your infra, id, and org-root accounts for last.
+* If you were using Atlantis to maintain a namespace, and you have this
+  component in your `terraform.tf` `backend` configuration:
+
+  ```text
+  role_arn       = "arn:aws:iam::123456789000:role/atlantis"
+  ```
+
+  Then before you begin tearing down that namespace, you need to remove
+  this line, run a `terraform apply`, and then do your `terraform destroy`.
+  If you do not do this, then your `destroy` will fail and the state will
+  become corrupted because you will destroy the role Terraform is using
+  to perform the destroy. Doing the `terraform apply` allows Terraform to
+  cleanly change the backend configuration first.
+* Similarly, when tearing down the `admin-global` namespace, you should
+  change your `.aws/config` profile for the account you're destroying to
+  use the `OrganizationAccountAccessRole` with the `org-root` account as
+  the `source_profile`. This prevents you from having the same problem
+  when your `admin` role is destroyed.
+* Do not destroy resources in the `admin-global` namespace until all
+  other namespaces for the account except the `bootstrap` namespace have
+  been torn down. Don't destroy `bootstrap` until `admin-global` has been
+  torn down.
+* When you are ready to tear down the `org-root` account, create an IAM
+  user *without* Terraform, give it credentials and attach an MFA, and
+  change your `org-root` profile to use those credentials. This will allow
+  you to cleanly tear down the entire `org-root` account safely, otherwise
+  you will delete the IAM user you are using to tear down the account in
+  the middle of the `destroy` and corrupt the state.
+* It's easiest to wait until all resources in all your accounts are
+  destroyed to begin closing accounts; this will remove all the
+  organization SCPs and let you remove accounts from the organization
+  as you go.
+* In order to completely close an account (aside from the `org-root`
+  account, you'll need to go through the password recovery procedure for
+  the root user account (ie, you need to try to log in with the email
+  address for the account and click the "Forgot Password" link), then
+  set up billing information for the account. Once you have done that,
+  you can safely remove the account from the AWS Organization (either
+  from the console or using the CLI) and then close it from the My
+  Account panel.
+
+## CircleCI
+
+Assuming you've used CircleCI for your CI/CD pipeline on the project,
+you should be able to just terminate your CircleCI support plan as soon
+as you don't need to do anymore deploys or validate code. You can do this
+from the "Project Settings" page on the CircleCI dashboard.
+
+## GitHub
+
+Once you've torn down your AWS infrastructure and CircleCI, you can
```

Comment:
why do we recommend destroying the whole github org? we often use tf to control who has access, so wouldn't it be enough to remove all trussels from the org?

Reply:
If we are paying for the organization, and the client doesn't want to take it over, it has to go somewhere. A GitHub organization isn't just a logical collection of people like a team, it's a paid product that owns repos and contains teams.

Reply:
I don't think you want a dangling org with no purpose.